src/he/ssl_ctx.h

```diff
@@ -470,6 +470,21 @@ bool he_ssl_ctx_is_auth_cb_set(he_ssl_ctx_t *ctx);
 void he_ssl_ctx_set_populate_network_config_ipv4_cb(
     he_ssl_ctx_t *ctx, he_populate_network_config_ipv4_cb_t pop_network_cb);
 
+/**
+ * @brief Sets te function that will be called when Lightway PMTUD changes state
+ * @param ctx A pointer to a valid SSL context
+ * @param pmtud_state_change_cb The function to be called when Lightway PMTUD changes state
+ */
+void he_ssl_ctx_set_pmtud_state_change_cb(he_ssl_ctx_t *ctx,
```

Comment:
Should we mark these 2 callbacks as optional if the client / server never intends to use the PMTUD feature (never inteds to call `he_conn_start_pmtu_discovery` and `he_conn_pmtud_probe_timeout`) ?

Reply:
Done.